TrackletAlgorithm/MemoryTemplateBinnedCM.h

```diff
@@ -150,30 +153,18 @@ class MemoryTemplateBinnedCM{
 
       if (nentry == ((1 << (NBIT_ADDR-NBIT_BIN)) - 1)) return false;
 
-	nentries_[ibx*kNBinsRZ+ibin].range(ireg*4+3,ireg*4)=nentry+1;
-	if (ibin!=0) {
-	  nentries_[ibx*kNBinsRZ+ibin-1].range((ireg+8)*4+3,(ireg+8)*4)=nentry+1;
-	}
-	binmask8_[ibx][ibin].set_bit(ireg,true);
+    	nentries_[ibx*kNBinsRZ+ibin].range(ireg*4+3,ireg*4)=nentry+1;
+    	if (ibin!=0) {
+	      nentries_[ibx*kNBinsRZ+ibin-1].range((ireg+8)*4+3,(ireg+8)*4)=nentry+1;
+    	}
+	    binmask8_[ibx][ibin].set_bit(ireg,true);
 
-	//icopy comparison must be signed int or future SW fails
-        writememloop:for (signed int icopy=0;icopy< (signed) NCP;icopy++) {
+    	//icopy comparison must be signed int or future SW fails
+      writememloop:for (signed int icopy=0;icopy< (signed) NCP;icopy++) {
 #pragma HLS unroll
-	  dataarray_[icopy][ibx][getNEntryPerBin()*slot+nentry] = data;
-	}
+	      dataarray_[icopy][ibx][getNEntryPerBin()*slot+nentry] = data;
+    	}
```

Comment:
@tomalin These changes just seem to affect the indentation, which was already a bit off, and now looks even more off, at least in my text editor. Could you clean this up, perhaps removing tabs altogether and consistently using two spaces to indent lines inside brackets?

Reply:
The problem is that the spaces in the HLS code are often created with "tab", and the number of spaces that "tab" corresponds to is not well-defined. It differs on different computers and in different editors/viewing tools. To fix this, we'd really have to globally replace all tabs by spaces, and then run clang to enforce a sensible indentation. And ask people to stop using tabs when editing the HLS code. But this should wait until there are no outstanding PRs.

Reply:
I completely agree, but I also think we can clean things up as we change other things. Especially with this block of code in particular, where the only changes that have been made are to the indentation, unless I've missed something.

Reply:
I can do this, but it would mean replacing all tabs with spaces in every line of every file where you've queried my changes to the indentation. And this of course would make it harder during your PR review to spot what I've changed. I suggest I make this change at the end, after I've answered all your non-indentation related comments?

Reply:
OK, these indentation comments are the only ones left. The block highlighted here is the most significant example, since the only changes you made, AFAICT, are to the indentation. Some of those changes also don't make sense, like lines 156 and 160 being in different columns now.

I included suggestions for the other blocks I highlighted that you can simply accept if you like, so I hope these comments are fairly trivial to address.